graphql-schemas/schema.yml

```diff
@@ -854,7 +864,55 @@
   - { name: vault_tls_secret_path, type: string }
   - { name: vault_tls_secret_version, type: int }
 
-- name: NamespaceTerraformResource_v1
+- name: NamespaceTerraformProviderResource_v1
```

Comment:
the name suggests that within this schema, one resource would be found, instead it contains all resources belonging to a certain provisioner. this is a change

Reply:
resolved by renaming to NamespaceTerraformProvisionProvider_v1